robot_calibration/src/finders/plane_finder.cpp

```diff
@@ -221,22 +224,48 @@ sensor_msgs::PointCloud2 PlaneFinder::extractPlane(sensor_msgs::PointCloud2& clo
     points(2, i) = (xyz + i)[Z];
   }
 
-  // Find centroid
-  Eigen::Vector3d centroid(points.row(0).mean(), points.row(1).mean(), points.row(2).mean());
-
-  // Center the cloud
-  points.row(0).array() -= centroid(0);
-  points.row(1).array() -= centroid(1);
-  points.row(2).array() -= centroid(2);
+  // Find the best fit plane
```

Comment:
alternatively, you can throw this into a helper function that:
- takes in: the cloud (or the xyz iterator and cloud size)
- returns: the best_normal vector